portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -14,19 +14,37 @@
 
 package com.google.sps.servlets;
 
+import com.google.sps.data.CommentData;
+import com.google.gson.Gson;
 import java.io.IOException;
 import javax.servlet.annotation.WebServlet;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
+import java.util.*;
 
-/** Servlet that returns some example content. TODO: modify this file to handle comments data */
+/** Servlet that returns some example comments. */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
+  private final List<String> comments = new ArrayList<String>(Arrays.asList("Hello!", "I hope everyone is doing well!", "Stay safe!"));
```

Comment:
Can this not be a list of CommentData, so you don't have to create a new List each time doGet is called?

Reply:
Thanks for the suggestion! This has been fixed in the newest version that does not take hard coded comments anymore.